lib/services/rpcbind.pm

```diff
@@ -39,13 +39,15 @@ sub config_service {
 }
 
 sub enable_service {
-    common_service_action('rpcbind',   $service_type, 'enable');
-    common_service_action($nfs_server, $service_type, 'enable');
+    common_service_action('rpcbind',        $service_type, 'enable');
+    common_service_action('rpcbind.socket', $service_type, 'enable');
+    common_service_action($nfs_server,      $service_type, 'enable');
 }
 
 sub start_service {
-    common_service_action('rpcbind',   $service_type, 'start');
-    common_service_action($nfs_server, $service_type, 'start');
+    common_service_action('rpcbind',        $service_type, 'start');
```

Comment:
Same.

Reply:
changed and verified  https://openqa.nue.suse.com/tests/6353095